app/controllers/species_list_controller.rb

```diff
@@ -485,7 +485,7 @@ def do_add_observations(species_list, query)
     # large species_lists, such as "Neotropical Fungi".
     # species_list.observation_ids += ids
     Observation.connection.insert(%(
```

Comment:
```suggestion
    SpeciesListObservation.insert_all(
      ids.map { |id| { observation_id: id, species_list_id: species_list.id } }
    )
```
This fixes the Codeclimate/Brakeman SQL injection issue:
>Possible SQL injection NEW
        #{ids.map { |id| "(#{id},#{species_list.id})" }.join(",")}
Severity: MinorFound in [app/controllers/species_list_controller.rb](https://codeclimate.com/github/MushroomObserver/mushroom-observer/app/controllers/species_list_controller.rb?from_sha=6f1e5ccb&to_sha=ce0aecf9) by [brakeman](https://docs.codeclimate.com/docs/brakeman)

Reply:
Yes! Nice one.